content/hi/docs/reference/glossary/customresourcedefinition.md

```diff
@@ -0,0 +1,19 @@
+---
+title: कस्टम संसाधन परिभाषा
+id: CustomResourceDefinition
+date: 2018-04-12
+full_link: /docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/
+short_description: >
+  कस्टम कोड जो एक संसाधन को परिभाषित करता है जिसे आपके Kubernetes API सर्वर में बिना पूरा कस्टम सर्वर बनाए जोड़ा जा सकता है।
+
+aka: 
+tags:
+- fundamental
+- operation
+- extension
+---
+ कस्टम कोड जो एक संसाधन को परिभाषित करता है जिसे आपके Kubernetes API सर्वर में बिना पूरा कस्टम सर्वर बनाए जोड़ा जा सकता है।
+ 
+<!--more-->
+
+कस्टम रिसोर्स डिफिनिशन आपको अपने वातावरण के लिए Kubernetes API का विस्तार करने की अनुमति देती हैं यदि सार्वजनिक रूप से समर्थित API संसाधन आपकी आवश्यकताओं को पूरा नहीं कर सकते।
```

Comment:
वातावरण -> एनवायरनमेंट

use this changes

Reply:
Sure. As per suggested replaced एनवायरनमेंट and कस्टम रिसोर्स डिफिनिशन. Pls Review once, Thanks!!